docs/decisions/0002-lerna-monorepo.rst

```diff
@@ -0,0 +1,108 @@
+################################
```

Comment:
Is there a reason we're using rst instead of `md` files here? (Like, don't change it, but I'm wondering why.)

Reply:
`.rst` seems to be the convention used elsewhere throughout edX repos for README files, so I'm just continuing with that convention here.